docs/releases/1.9.txt

```diff
@@ -356,6 +356,14 @@ Backwards incompatible changes in 1.9
     deprecation timeline for a given feature, its removal may appear as a
     backwards incompatible change.
 
+Paginator.page_range
```

Comment:
You are right. I should have mentioned in the commit message that the change to the iterator is done to fix both the inconsistency and the inefficiency of using a list. I'll amend the commit message and release notes.